doc/release-notes/0.18/0.18.md

```diff
@@ -0,0 +1,172 @@
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7406">#7406</a></td>
+<td valign="top">New <tt>exit</tt> dump agent and <tt>-XX:[+|-]ExitOnOutOfMemoryError</tt> option</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">The new <tt>exit</tt> dump agent saves information on the state of the VM when the VM shuts down, which can help you diagnose problems. For compatibility, OpenJ9 also supports the HotSpot option <tt>-XX:[+|-]ExitOnOutOfMemoryError</tt>. This option shuts down the VM when a <tt>java.lang.OutOfMemory</tt> error occurs by calling the <tt>exit</tt> dump agent.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/7332">#7332</a></td>
+<td valign="top">LUDCL caching enabled by default</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">This feature was disabled by default in Eclipse OpenJ9 0.17.0 due to issue <a href="https://github.com/eclipse/openj9/issues/7332">#7332</a>. This issue is resolved and the option is reenabled.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/7427">#7427</a></td>
+<td valign="top">Support for Terabyte suffixes for <tt>-X</tt> and <tt>-XX</tt> options</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">OpenJ9 now supports 't' and 'T' suffixes (indicating terabytes) for <tt>-X</tt> and <tt>-XX</tt> options that take a <tt>&lt;size&gt;</tt> parameter.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7728">#7728</a></td>
+<td valign="top">Extended platform support for pause-less garbage collection</td>
+<td valign="top">OpenJDK8 and later (macOS)</td>
+<td valign="top">Support for pause-less garbage collection (<tt>-Xgc:concurrentScavenge</tt>) is now extended to macOS systems.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7955">#7955</a></td>
+<td valign="top">Remove restriction on multi-layer shared cache</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">In the previous release, there is a restriction that the <tt>jvmtiSharedCacheInfo.isCorrupt</tt> field and the  <tt>SharedClassCacheInfo.isCacheCorrupt()</tt> method cannot detect a corrupted cache that has a layer number other than zero. This restriction is now removed.</td>
+</tr>
+
+</table>
+
+
+## Known Issues
+
+The v0.17 release contains the following known issues and limitations:
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1">
+<thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/7459">#7549</a></td>
```

Comment:
This one is resolved and can be removed.

Reply:
+1